PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/action/IntellijAzureActionManager.java

```diff
@@ -180,7 +180,7 @@ public void update(@Nonnull AnActionEvent e) {
             final Presentation presentation = e.getPresentation();
             Optional.ofNullable(view).ifPresent(v -> {
                 presentation.setText(v.getLabel());
-                presentation.setIcon(IntelliJAzureIcons.getIcon(v.getIconPath()));
```

Comment:
may also use FileUtil:exists

Reply:
not possible,  the icon path is a relative path.